lib/src/utils/constants.dart

```diff
@@ -79,6 +78,8 @@ const int p1StageDonation = 0x16;
 const int p1StageConfirm = 0x0a;
 const int p1StageWitnesses = 0x0f;
 
+const String maxUint64Str = '18446744073709551615';
```

Comment:
I think it's better to set it as BigInt instead. 

P.s. You'll need to use `final` instead of `const` for BigInt. But that's not going to cause any issues/differences in using it.